README.md

```diff
@@ -1,10 +1,15 @@
 ﻿# Parso Java library
+## Parso 2.0.3
```

Comment:
We haven't yet released version 2.0.2 to maven repo, so this changes can be added to version 2.0.2.